src/core/solvers/ddp.cpp

```diff
@@ -324,24 +340,31 @@ void SolverDDP::forwardPass(const double steplength) {
   if (raiseIfNaN(cost_try_)) {
     throw_pretty("forward_error");
   }
+  STOP_PROFILER("SolverDDP::forwardPass");
 }
 
 void SolverDDP::computeGains(const std::size_t t) {
+  START_PROFILER("SolverDDP::computeGains");
   const std::size_t nu = problem_->get_runningModels()[t]->get_nu();
   if (nu > 0) {
+    START_PROFILER("SolverDDP::Quu_inv");
     Quu_llt_[t].compute(Quu_[t].topLeftCorner(nu, nu));
+    STOP_PROFILER("SolverDDP::Quu_inv");
     const Eigen::ComputationInfo& info = Quu_llt_[t].info();
     if (info != Eigen::Success) {
       throw_pretty("backward_error");
     }
     K_[t].topRows(nu).noalias() = Qxu_[t].leftCols(nu).transpose();
 
     Eigen::Block<Eigen::MatrixXd> K = K_[t].topRows(nu);
+    START_PROFILER("SolverDDP::Quu_solve");
```

Comment:
We have two solveInPlace functions. It would be nice to include both like: 'Quuinv_Qux'

Reply:
I've included only the Quu_inv*Qux because I'm currently profiling only the cubic-complexity operations. I can change the name as you suggested